### PR TITLE
Add new collaborators command and printer

### PR DESCRIPTION
**Pull request by cored**

### CODE REVIEW

File: lib/octonaut/printers/repositories.rb [hunk 1]
```diff
@@ -33,6 +33,7 @@ def print_repo_table(repo, options = {})
       end
 
       def print_repos(repos, options = {})
+        puts caller.inspect
```
Comment (pengwynn) [inline lib/octonaut/printers/repositories.rb]: Oops.

**Comment by pengwynn** [comment]
Looks good! I've added you as a collaborator.

Before we get too far, we might want to bump to Octokit 2.0 and think about VCR/VCR for fixtures like we do over there.

**Comment by cored** [comment]
Sounds good to me, I will add two issues for both things.

On 10 September 2013 18:29, Wynn Netherland notifications@github.comwrote:

> Looks good! I've added you as a collaborator.
> 
> Before we get too far, we might want to bump to Octokit 2.0 and think
> about VCR/VCR for fixtures like we do over there.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/pengwynn/octonaut/pull/15#issuecomment-24199936
> .

## 

Rafael George

**Comment by cored** [comment]
@pengwynn Do you want me to wait till I upgrade Octokit to 2.x or can I just merge this pull request?

**Comment by pengwynn** [comment]
Merge away!

**Pull request merged**